datahub-web-react/src/app/entity/shared/constants.ts

```diff
@@ -47,4 +47,12 @@ export const EMPTY_MESSAGES = {
         title: 'No domain set',
         description: 'Group related entities based on your organizational structure using by adding them to a Domain.',
     },
+    contains: {
+        title: 'No contain yet',
```

Comment:
Contains no Terms

Terms can contain other terms to represent an "Has A" style relationship